src/routes/mail.routes.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from "express";
+import { getEmailsFromGmailHandler } from "../controllers/mail.controller";
+import deserializeUser from "../middleware/deserializeUser.middleware";
+import requireSameUser from "../middleware/requireSameUser.middleware";
+
+const mailRouter = Router();
+
+mailRouter.get(
+	"/mail/:id/gmail/:email",
+	deserializeUser,
+	requireSameUser,
+	getEmailsFromGmailHandler,
```

Comment:
Do we have to validate the Request schema against `GetEmailsFromGmailSchema`?

Reply:
Yup forgot